src/main/java/com/idrsolutions/microservice/utils/SettingsValidator.java

```diff
@@ -5,17 +5,18 @@
 import java.util.Map;
 import java.util.Set;
 
+/**
+ * An abstract class used to validate the settings that will be passed into a conversion.
+ * SettingsValidator will notify you if unexpected parameters are passed,
```

Comment:
This class is no abstract, can we update this line please.

Reply:
I can see how that could cause confusion, I think I was intending for generic. Does that definition line up with your understanding?

Reply:
> This class is no abstract, can we update this line please.

Updated in latest commit